### PR TITLE
removed duplicates

### PR DESCRIPTION
**Pull request by sourabh1031**
fixed duplicacy in the code

### CODE REVIEW

**Comment by danthareja** [comment]
> ### Yay, a pull request!
>  After you submit a pull request, one of the following will happen:
> 
> #### :sob: You don’t get a response. :sob:
> 
> Even on an active project, it’s possible that your pull request won’t get an immediate response. You should expect some delay as most open source maintainers do so in their free time and can be busy with other tasks. 
> 
> If you haven’t gotten a response in over a week, it’s fair to politely respond in the same thread, asking someone for a review. If you know the handle of the right person to review your pull request, you can @-mention them to send them a notification. Avoid reaching out to that person privately; remember that public communication is vital to open source projects.
> 
> If you make a polite bump and still nobody responds, it’s possible that nobody will respond, ever. It’s not a great feeling, but don’t let that discourage you. It’s happened to everyone! There are many possible reasons why you didn’t get a response, including personal circumstances that may be out of your control. Try to find another project or way to contribute. If anything, this is a good reason not to invest too much time in making a pull request before other community members are engaged and responsive.
> 
> #### :construction: You're asked to make changes to your pull request. :construction:
> 
> It’s very common that someone will request changes on your pull request, whether that’s feedback on the scope of your idea, or changes to your code. Often a pull request is just the start of the conversation.
> 
> When someone requests changes, be responsive. They’ve taken the time to review your pull request. Opening a PR and walking away is bad form. If you don’t know how to make changes, research the problem, then ask for help if you need it.
> 
> If you don’t have time to work on the issue anymore (for example, if the conversation has been going on for months, and your circumstances have changed), let the maintainer know so they’re not expecting a response. Someone else may be happy to take over.
>
> #### :-1: Your pull request doesn’t get accepted.  :-1:
> 
> It's possible your pull request may or may not be accepted in the end. If you’re not sure why it wasn’t accepted, it’s perfectly reasonable to ask the maintainer for feedback and clarification. Ultimately, however, you’ll need to respect that this is their decision. Don’t argue or get hostile. You’re always welcome to fork and work on your own version if you disagree!
>
> #### :tada: Your pull request gets accepted and merged. :tada:
> 
> Hooray! You’ve successfully made an open source contribution!
---
Thank you for the submission, @sourabh1031! I'll review your code shortly, hang tight.

File: src/calculator.js [hunk 1]
```diff
@@ -1,47 +1,31 @@
 exports._check = () => {
   // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule

File: src/calculator.js [hunk 2]
```diff
@@ -1,47 +1,31 @@
...
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
```
Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule

File: src/calculator.js [hunk 3]
```diff
@@ -1,47 +1,31 @@
...
   }
+
```
Comment (danthareja) [inline src/calculator.js]: Trailing spaces not allowed.

###### See http://eslint.org/docs/rules/no-trailing-spaces for details about this rule

File: src/calculator.js [hunk 4]
```diff
@@ -1,47 +1,31 @@
...
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+ 
+};
```
Comment (danthareja) [inline src/calculator.js]: Block must not be padded by blank lines.

###### See http://eslint.org/docs/rules/padded-blocks for details about this rule

File: src/calculator.js (hunk 1 above)
Comment (sourabh1031) [reply src/calculator.js]: i am not getting it .According to me we need to define global declaration of variables isn't?

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 5]
```diff
@@ -1,48 +1,31 @@
...
 exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule

File: src/calculator.js [hunk 6]
```diff
@@ -1,48 +1,31 @@
...
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
```
Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule

File: src/calculator.js [hunk 7]
```diff
@@ -1,48 +1,31 @@
...
 exports.divide = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check();
   return x / y;
 };
```
Comment (danthareja) [inline src/calculator.js]: Too many blank lines at the end of file. Max of 1 allowed.

###### See http://eslint.org/docs/rules/no-multiple-empty-lines for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 8]
```diff
@@ -1,48 +1,29 @@
...
 exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule

File: src/calculator.js [hunk 9]
```diff
@@ -1,48 +1,29 @@
...
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
```
Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 10]
```diff
@@ -1,48 +1,31 @@
+module.exports = exports;
+var x;
```
Comment (danthareja) [inline src/calculator.js]: All &#39;var&#39; declarations must be at the top of the function scope.

###### See http://eslint.org/docs/rules/vars-on-top for details about this rule

Comment (danthareja) [inline src/calculator.js]: Unexpected var, use let or const instead.

###### See http://eslint.org/docs/rules/no-var for details about this rule

File: src/calculator.js [hunk 11]
```diff
@@ -1,48 +1,31 @@
...
+var y;
```
Comment (danthareja) [inline src/calculator.js]: All &#39;var&#39; declarations must be at the top of the function scope.

###### See http://eslint.org/docs/rules/vars-on-top for details about this rule

Comment (danthareja) [inline src/calculator.js]: Unexpected var, use let or const instead.

###### See http://eslint.org/docs/rules/no-var for details about this rule

File: src/calculator.js [hunk 12]
```diff
@@ -1,48 +1,31 @@
...
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+};
+
+exports.add = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

File: src/calculator.js [hunk 13]
```diff
@@ -1,48 +1,31 @@
...
+  exports._check();
   return x + y;
 };
 
 exports.subtract = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

File: src/calculator.js [hunk 14]
```diff
@@ -1,48 +1,31 @@
...
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check();
   return x - y;
 };
 
 exports.multiply = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

File: src/calculator.js [hunk 15]
```diff
@@ -1,48 +1,31 @@
...
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check();
   return x * y;
 };
 
 exports.divide = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 16]
```diff
@@ -1,48 +1,31 @@
...
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+};
+
+exports.add = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

File: src/calculator.js [hunk 17]
```diff
@@ -1,48 +1,31 @@
...
+  exports._check();
   return x + y;
 };
 
 exports.subtract = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

File: src/calculator.js [hunk 18]
```diff
@@ -1,48 +1,31 @@
...
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check();
   return x - y;
 };
 
 exports.multiply = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

File: src/calculator.js [hunk 19]
```diff
@@ -1,48 +1,31 @@
...
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check();
   return x * y;
 };
 
 exports.divide = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 20]
```diff
@@ -1,48 +1,23 @@
...
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+};
+exports.add = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 21]
```diff
@@ -1,48 +1,23 @@
...
   return x + y;
 };
-
 exports.subtract = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 22]
```diff
@@ -1,48 +1,23 @@
...
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x - y;
 };
-
 exports.multiply = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 23]
```diff
@@ -1,48 +1,23 @@
...
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x * y;
 };
-
 exports.divide = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: &#39;x&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: &#39;y&#39; is already declared in the upper scope.

###### See http://eslint.org/docs/rules/no-shadow for details about this rule

Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 24]
```diff
@@ -1,48 +1,23 @@
...
+  let x;
+  let y;
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+};
+exports.add = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 25]
```diff
@@ -1,48 +1,23 @@
...
   return x + y;
 };
-
 exports.subtract = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 26]
```diff
@@ -1,48 +1,23 @@
...
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x - y;
 };
-
 exports.multiply = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 27]
```diff
@@ -1,48 +1,23 @@
...
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x * y;
 };
-
 exports.divide = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 28]
```diff
@@ -1,48 +1,23 @@
...
-exports.add = (x, y) => {
+  let x;
+  let y;
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+}
```
Comment (danthareja) [inline src/calculator.js]: Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule

File: src/calculator.js [hunk 29]
```diff
@@ -1,48 +1,23 @@
...
+exports.add = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 30]
```diff
@@ -1,48 +1,23 @@
...
   return x + y;
-};
-
+}
```
Comment (danthareja) [inline src/calculator.js]: Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule

File: src/calculator.js [hunk 31]
```diff
@@ -1,48 +1,23 @@
...
 exports.subtract = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 32]
```diff
@@ -1,48 +1,23 @@
...
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x - y;
-};
-
+}
```
Comment (danthareja) [inline src/calculator.js]: Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule

File: src/calculator.js [hunk 33]
```diff
@@ -1,48 +1,23 @@
...
 exports.multiply = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 34]
```diff
@@ -1,48 +1,23 @@
...
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x * y;
-};
-
+}
```
Comment (danthareja) [inline src/calculator.js]: Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule

File: src/calculator.js [hunk 35]
```diff
@@ -1,48 +1,23 @@
...
 exports.divide = (x, y) => {
```
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js [hunk 36]
```diff
@@ -1,48 +1,23 @@
...
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
   return x / y;
-};
-
-module.exports = exports;
+}
```
Comment (danthareja) [inline src/calculator.js]: Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js (hunk 24 above)
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js (hunk 25 above)
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js (hunk 26 above)
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

File: src/calculator.js (hunk 27 above)
Comment (danthareja) [inline src/calculator.js]: Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 37]
```diff
@@ -1,48 +1,23 @@
...
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+module.exports = exports;
+exports._check = () => ({
+  let x;
```
Comment (danthareja) [inline src/calculator.js]: Parsing error: The keyword &#39;let&#39; is reserved

###### See http://eslint.org/docs/rules/ for details about this rule

**Comment by danthareja** [comment]
> :sparkles: Changes synced :sparkles:
>
> When a pull request is open, GitHub tracks changes to the branch it was opened from. As you push changes to your fork's `patch-1` branch, you'll automatically see them here.

I'm reviewing your changes now, @sourabh1031. Let's get this merged soon!

File: src/calculator.js [hunk 38]
```diff
@@ -1,48 +1,23 @@
...
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+module.exports = exports;
+exports._check = () => ({
+  var x;
```
Comment (danthareja) [inline src/calculator.js]: Parsing error: Unexpected keyword &#39;var&#39;

###### See http://eslint.org/docs/rules/ for details about this rule

**Pull request closed without merging**